### PR TITLE
[14.0][FIX] Add sudo() on account.asset.line

### PR DESCRIPTION
**Pull request by newtratip**
This PR is fix for error **"You are not allowed to access 'Asset depreciation table line' (account.asset.line) records."**
Selection_438

cc: @kittiu

### CODE REVIEW

File: account_asset_management/models/account_move.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ class AccountMove(models.Model):
     def _compute_asset_count(self):
         for rec in self:
             assets = (
-                self.env["account.asset.line"]
+                self.sudo().env["account.asset.line"]
```
Comment (kittiu) [inline account_asset_management/models/account_move.py]:
```suggestion
                self.env["account.asset.line"].sudo()
```

**Comment by kittiu** [comment]
Please check pre-commit

File: account_asset_management/models/account_move.py (hunk 1 above)
Comment (newtratip) [reply account_asset_management/models/account_move.py]: fixed.

**Comment by AaronHForgeFlow** [comment]
Is it possible to add the attribute "groups" in the field in the view instead of doing this?

**Comment by kittiu** [comment]
@newtratip , I think what @AaronHForgeFlow say make more sense.
In fact, the error you face, was it from your custom module?

**Comment by newtratip** [comment]
Yes, it makesense, On view can already access only group Invoicing / Billing https://github.com/OCA/account-financial-tools/blob/14.0/account_asset_management/views/account_move.xml#L7

This may be my mistake, I will close it.

**Pull request closed without merging**